plugins/woocommerce/src/Blocks/BlockTemplatesController.php

```diff
@@ -28,76 +28,73 @@ public function init() {
-
-		if ( wc_current_theme_is_fse_theme() ) {
-			// By default, the Template Part Block only supports template parts that are in the current theme directory.
-			// This render_callback wrapper allows us to add support for plugin-housed template parts.
-			add_filter(
-				'block_type_metadata_settings',
-				function ( $settings, $metadata ) {
-					if (
-						isset( $metadata['name'], $settings['render_callback'] ) &&
-						'core/template-part' === $metadata['name'] &&
-						in_array( $settings['render_callback'], array( 'render_block_core_template_part', 'gutenberg_render_block_core_template_part' ), true )
-					) {
-						$settings['render_callback'] = array( $this, 'render_woocommerce_template_part' );
-					}
-					return $settings;
-				},
-				10,
-				2
-			);
-
-			// Prevents shortcodes in templates having their HTML content broken by wpautop.
-			// @see https://core.trac.wordpress.org/ticket/58366 for more info.
-			add_filter(
-				'block_type_metadata_settings',
-				function ( $settings, $metadata ) {
-					if (
-						isset( $metadata['name'], $settings['render_callback'] ) &&
-						'core/shortcode' === $metadata['name']
-					) {
-						$settings['original_render_callback'] = $settings['render_callback'];
-						$settings['render_callback']          = function ( $attributes, $content ) use ( $settings ) {
-							// The shortcode has already been rendered, so look for the cart/checkout HTML.
-							if ( strstr( $content, 'woocommerce-cart-form' ) || strstr( $content, 'wc-empty-cart-message' ) || strstr( $content, 'woocommerce-checkout-form' ) ) {
-								// Return early before wpautop runs again.
-								return $content;
-							}
-
-							$render_callback = $settings['original_render_callback'];
-
-							return $render_callback( $attributes, $content );
-						};
-					}
-					return $settings;
-				},
-				10,
-				2
-			);
-
-			/**
-			 * Prevents the pages that are assigned as cart/checkout from showing the "template" selector in the page-editor.
-			 * We want to avoid this flow and point users towards the site editor instead.
-			 */
-			add_action(
-				'current_screen',
-				function () {
-					if ( ! is_admin() ) {
-						return;
-					}
-
-					$current_screen = get_current_screen();
-
-					// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-					if ( $current_screen && 'page' === $current_screen->id && ! empty( $_GET['post'] ) && in_array( absint( $_GET['post'] ), array( wc_get_page_id( 'cart' ), wc_get_page_id( 'checkout' ) ), true ) ) {
-						wp_add_inline_style( 'wc-blocks-editor-style', '.edit-post-post-template { display: none; }' );
-					}
-				},
-				10
-			);
-		}
+
+		// By default, the Template Part Block only supports template parts that are in the current theme directory.
+		// This render_callback wrapper allows us to add support for plugin-housed template parts.
+		add_filter(
+			'block_type_metadata_settings',
+			function ( $settings, $metadata ) {
```

Comment:
Just curious as to why these are inline functions instead of methods?

Reply:
I don't think there is any specific _why_ other than historical reasons. I'm happy to change that to make it consistent with the rest of the class, but would rather do it in a separate PR to avoid mixing this PR with unrelated changes. How does it sound?

Reply:
>but would rather do it in a separate PR to avoid mixing this PR with unrelated changes. How does it sound?

I think thats fine, using named methods will help give the reader an idea of its intent without having to read the code as well as make it more consistent.